Header header

float32 voltage
float32 current
float32 cpu_load
